.github/workflows/create-release-pr.yml

```diff
@@ -0,0 +1,26 @@
+# Automatically creates a PR to development for every new `release-` branch
+name: Create Release PR
+
+on: create
```

Comment:
Thanks for the PR! Are you certain this condition will only run for new Mudlet releases?

Reply:
The `create` event does not support filtering (like `push` does). So this workflow will be called for all new branches. That's why we need the condition on line 12 (which I just noticed I forgot to uncomment 🤦🏻)

`if: startsWith(github.ref_name, 'release-')` only allow the job to be executed when the name of the created branch **or tag** starts with `"release-"`